system-test/system.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+'use strict';
```

Comment:
I'm starting to think that if we have sample tests, we don't need a system test for gapics.  What do you think?

Reply:
yeah, it is a little redundant, our sample tests are essentially a system test.

the world I would like, honestly, would be that we collect coverage for `src/`, and we have enough sample tests that almost all of our generated API surface has coverage. this is why I removed `src/**` from ignore rules for coverage.